runtime/interpreter/value_test.go

```diff
@@ -626,3 +626,11 @@ func TestVisitor(t *testing.T) {
 	require.Equal(t, 1, intVisits)
 	require.Equal(t, 1, stringVisits)
 }
+
+func TestBlockValue(t *testing.T) {
+	var block BlockValue = BlockValue{ 4, 5, &ArrayValue{}, 5.0}
+	// static type test
+	var actualTs UFix64Value = block.Timestamp
+	const expectedTs UFix64Value = 5.0
+	assert.Equal(t, expectedTs, actualTs)
```

Comment:
This test case doesn't really test functionality of the code base, as it constructs the right value (a `BlockValue` is created with the `5.0` literal in the first line). 

Maybe replace this with a test of `runtime.NewBlockValue`

Reply:
Added this test to statically validate the type of the `block.Timestamp` field. (Changing the type of the field will fail the compilation of this test)

The assert is probably redundant, it just checks the type and value of the timestamp, against the expected type and value.